tests/sqlparser_mssql.rs

```diff
@@ -192,6 +192,35 @@ fn parse_mssql_apply_join() {
     );
 }
 
+#[test]
+fn parse_mssql_cross_apply_json() {
```

Comment:
Could we add one scenario that checks the returned AST (i.e that we get a TableFactor::OpenJSON containing the intended components of the input)?